# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/ne/move-up-down.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/inputclass.c.orig	Sat Sep  1 00:32:32 2001
+++ ./src/inputclass.c	Thu Dec 13 21:39:57 2001
@@ -108,15 +108,15 @@
 	/* The following bindings are for the terminfo extended codes (see keycodes.h). */
 
 	/* Cursor movement keys */
-	NULL, LINEUP_ABBREV, LINEDOWN_ABBREV, MOVELEFT_ABBREV, MOVERIGHT_ABBREV, MOVEINCUP_ABBREV, MOVEINCDOWN_ABBREV, NEXTPAGE_ABBREV,
-	PREVPAGE_ABBREV, LINEDOWN_ABBREV, LINEUP_ABBREV, NULL, NULL, NULL, NULL, NULL,
+	NULL, LINEUP_ABBREV, LINEDOWN_ABBREV, MOVELEFT_ABBREV, MOVERIGHT_ABBREV, MOVEINCUP_ABBREV, MOVEINCDOWN_ABBREV, PICK(PAGEDOWN_ABBREV, NEXTPAGE_ABBREV),
+	PICK(PAGEUP_ABBREV, PREVPAGE_ABBREV), LINEDOWN_ABBREV, LINEUP_ABBREV, NULL, NULL, NULL, NULL, NULL,
 
 	/* Editing keys */
 	DELETEEOL_ABBREV, NULL, BACKSPACE_ABBREV, DELETELINE_ABBREV, UNDELLINE_ABBREV, DELETECHAR_ABBREV, INSERT_ABBREV, NULL,
 	CLEAR_ABBREV, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
 
 	/* Keypad keys and fake (simulated) menu key */
-	MOVESOF_ABBREV, PREVPAGE_ABBREV, TOGGLESEOL_ABBREV, MOVEEOF_ABBREV, NEXTPAGE_ABBREV, EXEC_ABBREV, NULL, NULL,
+	MOVESOF_ABBREV, PICK(PAGEUP_ABBREV, PREVPAGE_ABBREV), TOGGLESEOL_ABBREV, MOVEEOF_ABBREV, PICK(PAGEDOWN_ABBREV, NEXTPAGE_ABBREV), EXEC_ABBREV, NULL, NULL,
 	NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
 
 	NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
--- ./src/menu.c.orig	Sat Sep  1 00:32:32 2001
+++ ./src/menu.c	Thu Dec 13 21:39:57 2001
@@ -182,10 +182,10 @@
 		{ "Line Down       ", LINEDOWN_ABBREV },
 PICK(   "Prev Page     ^P", PREVPAGE_ABBREV    , "Page Up       ^P", PAGEUP_ABBREV)
 PICK(   "Next Page     ^N", NEXTPAGE_ABBREV    , "Page Down     ^N", PAGEDOWN_ABBREV)
-PICK(   "Page Up         ", PAGEUP_ABBREV      , "Prev Page       ", PREVPAGE_ABBREV)
-PICK(   "Page Down       ", PAGEDOWN_ABBREV    , "Next Page       ", NEXTPAGE_ABBREV)
-		{ "Start Of File [A", MOVESOL_ABBREV },
-		{ "End Of File   [E", MOVEEOL_ABBREV },
+PICK(   "Page Up     PgUp", PAGEUP_ABBREV      , "Prev Page   PgUp", PREVPAGE_ABBREV)
+PICK(   "Page Down   PgDn", PAGEDOWN_ABBREV    , "Next Page   PgDn", NEXTPAGE_ABBREV)
+		{ "Start Of File [A", MOVESOF_ABBREV },
+		{ "End Of File   [E", MOVEEOF_ABBREV },
 		{ "Start Of Line ^A", MOVESOL_ABBREV },
 		{ "End Of Line   ^E", MOVEEOL_ABBREV },
 		{ "Top Of Screen   ", MOVETOS_ABBREV },
--- ./src/navigation.c.orig	Sat Sep  1 00:32:32 2001
+++ ./src/navigation.c	Thu Dec 13 21:42:43 2001
@@ -961,7 +961,7 @@
 		if (b->cur_y == ne_lines-2) move_to_bof(b);
 		else next_page(b);
 	}
-	else move_to_eol(b);
+	move_to_eol(b);
 }
 
 /* Same as above, towards the top. */
